controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -401,7 +401,31 @@ bool master_thread::autoconfig_wsc_add_m2(std::shared_ptr<ieee1905_1::tlvWscM1>
             ? WSC::FRONTHAUL_BSS
             : (bss_type & WSC::BACKHAUL_BSS ? WSC::BACKHAUL_BSS : WSC::TEARDOWN);
     tlvWscM2->primary_device_type_attr().sub_category_id = WSC::WSC_DEV_NETWORK_INFRA_GATEWAY;
-    // TODO: Finalize with encryption
+    // Encrypted settings
+    auto encrypted_settings = tlvWscM2->create_encrypted_settings();
```

Comment:
can this commit work without the autogenerated from the next commit?

Reply:
Yes since the build will generate whatever is missing.